src/main/javascript/crypto/src/libs/AttestedObject.ts

```diff
@@ -21,16 +15,20 @@ export class AttestedObject {
     private unsignedEncoding: string;
     private derEncodedProof: string;
     private signature: string;
-    constructor(private attestableObject: Cheque, private att: SignedAttestation,
-                // private keys: KeyPair,
-                private attestationSecret: bigint ,private chequeSecret: bigint) {
+    constructor(
+        private attestableObject: AttestableObject,
+        private att: SignedAttestation,
+        // private keys: KeyPair,
+        private attestationSecret: bigint ,
+        private chequeSecret: bigint
```

Comment:
I think this should just be called objectSecret now

Reply:
renamed